hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -714,6 +714,24 @@ def resolve_List(thing):
     return st.lists(st.from_type(thing.__args__[0]))
 
 
+try:
+    import numpy.typing as npt
+
+    from numpy._typing._generic_alias import ScalarType
+    from hypothesis.extra.numpy import array_shapes, arrays, scalar_dtypes
```

Comment:
- The import order is still an issue here - run `shed` instead of `black` 😉 
- use try/except/else, and put the function definition in the else block.

Reply:
sure, forgot about it